.goreleaser.yaml

```diff
@@ -0,0 +1,54 @@
+before:
+  hooks:
+    - make generate
+
+builds:
+  - id: flow-cli
+    main: ./cmd/flow
+    goos:
+      - darwin
+      - linux
+    goarch:
+      - amd64
+      - arm64
+    env:
+      - CGO_FLAGS="-O2 -D__BLST_PORTABLE__"
+      - CGO_ENABLED=1
+      - CC_darwin_amd64=o64-clang
+      - CXX_darwin_amd64=o64-clang+
+      - CC_darwin_arm64=oa64-clang
+      - CXX_darwin_arm64=oa64-clang++
+      - CC_linux_amd64=x86_64-linux-gnu-gcc
+      - CXX_linux_amd64=x86_64-linux-gnu-g++
+      - CC_linux_arm64=aarch64-linux-gnu-gcc
+      - CXX_linux_arm64=aarch64-linux-gnu-g++
+      - CC_windows_amd64=x86_64-w64-mingw32-gcc
+      - CXX_windows_amd64=x86_64-w64-mingw32-g++
+      - CC_windows_arm64=aarch64-w64-mingw32-gcc
+      - CXX_windows_arm64=aarch64-w64-mingw32-g++
+      - 'CC={{ index .Env (print "CC_" .Os "_" .Arch) }}'
```

Comment:
Nice solution!
I'm curious where do `Os` and `Arch` come from? are they equal to `goos` and `goarch` from above?

Reply:
Yeah it's something given to us from goreleaser coming from the goos/goarch above 👍